circuits/src/zk_gadgets/nonnative.rs

```diff
@@ -882,6 +894,159 @@ impl NonNativeElementVar {
     }
 }
 
+// -----------------------
+// | Secret Shared Types |
+// -----------------------
+
+/// Represents an additive secret share of a non-native element
+#[derive(Clone, Debug)]
+pub struct NonNativeSecretShare {
+    /// The underlying BigUint
+    pub(crate) val: BigUint,
+    /// The modulus of the field this element is defined over
+    pub(crate) field_mod: FieldMod,
+}
+
+/// Represents an additive secret share of a non-native element
```

Comment:
nit: not immediately clear what the difference is between this and the `NonNativeSecretShare` struct above... could we make this comment different?

Reply:
Oops, moving too fast and I duplicated it, remove the one above. Thanks!